### DISCUSSION TITLE
Using nvim-metals with Scala Version 2.10.3: How to Get Started?

### DISCUSSION THREAD

**Discussion by meetorion** [scalameta/metals#5554] [Q&A] [answered]
I have a project written in Scala 2.10.3, and I am unable to use the latest nvim-metals for importing the build. How can I proceed to use nvim-metals with my Scala 2.10.3 project, considering I do not have permission to upgrade the Scala version used in the project?

**Comment by ckipp01** [comment, marked as answer]
Hey @meetorion, I transferred this over to the main Metals repo since this isn't unique to nvim-metals. Metals unfortunately doesn't support Scala 2.10 as it's an incredibly old Scala version. If you aren't able to upgrade the version at all you may have better luck trying with IntelliJ.

**Reply by meetorion** [reply]
thanks
